src/vaadin-avatar-group.html

```diff
@@ -33,14 +36,48 @@
       }
     </style>
     <div id="container">
-      <template is="dom-repeat" items="[[items]]">
-        <vaadin-avatar name="[[item.name]]" abbr="[[item.abbr]]" img="[[item.img]]" part="avatar"></vaadin-avatar>
+      <template is="dom-repeat" items="[[__computeItems(items.*, max)]]">
+        <vaadin-avatar
+          name="[[item.name]]"
+          abbr="[[item.abbr]]"
+          img="[[item.img]]"
+          part="avatar"
+          theme$="[[theme]]"
+        ></vaadin-avatar>
       </template>
+      <vaadin-avatar
```

Comment:
nit: should it have different title? not `anonymous`? 🤔

Reply:
Thanks, fixed.